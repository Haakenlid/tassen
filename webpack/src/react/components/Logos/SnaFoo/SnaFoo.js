const SnaFoo = ({  }) => (
  <div className="SnaFoo">
    
  </div>
)

export default SnaFoo
