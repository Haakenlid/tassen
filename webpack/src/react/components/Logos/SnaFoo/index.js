import SnaFoo from './SnaFoo.js'
import './SnaFoo.scss'
export default SnaFoo
