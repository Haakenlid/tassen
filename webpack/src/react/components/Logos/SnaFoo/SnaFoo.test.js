import SnaFoo from './index'
import renderer from 'react-test-renderer'

describe('SnaFoo', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<SnaFoo />).toJSON()
    expect(tree).toEqual(expect.any(Object))
    expect(tree).toMatchSnapshot()
  })
})
