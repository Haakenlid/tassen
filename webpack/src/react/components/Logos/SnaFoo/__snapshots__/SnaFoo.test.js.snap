// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnaFoo renders correctly 1`] = `
<div
  className="SnaFoo"
/>
`;
